movie-recommendations/src/components/Movie.css

```diff
@@ -0,0 +1,71 @@
+.MovieContainer{
+    height: 207px;
+    width: 150px;
+    background-color: #Aea1a1;
+    border-radius: 15px;
+    cursor: pointer;
+    display: flex;
+    position: relative;
+
+}
+
+
+.MovieContainer:hover img{
+    opacity: 0.5;
+}
+.MovieContainer:hover .icons{
+    opacity: 1;
+}
+
+.MovieContainer img{
+    height: 100%;
+    width: 100%;
+    border-radius: inherit;
+    object-fit: cover;
+    position: relative;
+    z-index: 1;
+    transition: opacity 100ms ease-in-out;
+
+}
+
+.MovieContainer div{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    position: relative;
+    justify-content: center;
+    z-index: 999;
+    align-items: center;
+    
+}
+.MovieContainer .icons{
+    display: flex;
+    position: absolute;
+    z-index: 999;
+    opacity: 0;
+    transition: opacity 100ms ease-in-out;
+
+}
+
+.MovieContainer .icon{
+    padding: 0;
+    padding-bottom: 10px;
```

Comment:
the last icon "the link icon" will get bottom padding of 10px that will make the icons not centered in the poster